src/issuer/SsiCredentialIssuer.Service/Controllers/IssuerController.cs

```diff
@@ -42,16 +42,18 @@ public static RouteGroupBuilder MapIssuerApi(this RouteGroupBuilder group)
     {
         var issuer = group.MapGroup("/issuer");
 
-        issuer.MapGet("useCaseParticipation", (IIssuerBusinessLogic logic) => logic.GetUseCaseParticipationAsync())
+        issuer.MapGet("useCaseParticipation", (IIssuerBusinessLogic logic,
+                [FromQuery(Name = "status")] string? status) => logic.GetUseCaseParticipationAsync(status))
```

Comment:
Can be changed to StatusType as well

```suggestion
                [FromQuery(Name = "status")] StatusType? status) => logic.GetUseCaseParticipationAsync(status))
```

Reply:
The same here, it is just to have a customized message.